### PR TITLE
use /etc not /run for etcd dropin

### PR DESCRIPTION
**Pull request by robszumski**
Fixes https://github.com/coreos/bugs/issues/292

cc: @marineam

### CODE REVIEW

File: distributed-configuration/customize-etcd-unit/index.md [hunk 1]
```diff
@@ -17,7 +16,7 @@ etcd supports client certificates as a way to provide secure communication betwe
 
 This site has a [good reference for how to generate self-signed key pairs](http://www.g-loaded.eu/2005/11/10/be-your-own-ca/) or you could use [etcd-ca](https://github.com/coreos/etcd-ca) to generate certs and keys. 
 
-We need to create our drop-in unit in `/run/systemd/system/etcd.service.d/`. If you run `systemctl status etcd` you can see that CoreOS is already generating a few drop-in units for etcd as part of the OEM and cloudinit processes. To ensure that our drop-in runs after these, we name it `30-certificates.conf`
+We need to create our drop-in unit in `/etcd/systemd/system/etcd.service.d/`. If you run `systemctl status etcd` you can see that CoreOS is already generating a few drop-in units for etcd as part of the OEM and cloudinit processes. To ensure that our drop-in runs after these, we name it `30-certificates.conf` and place them in `/etc/`.
```
Comment (crawford) [inline distributed-configuration/customize-etcd-unit/index.md]: etc

Comment (robszumski) [inline distributed-configuration/customize-etcd-unit/index.md]: Autocorrect on my mac....

File: distributed-configuration/customize-etcd-unit/index.md [hunk 2]
```diff
@@ -17,7 +16,7 @@ etcd supports client certificates as a way to provide secure communication betwe
 
 This site has a [good reference for how to generate self-signed key pairs](http://www.g-loaded.eu/2005/11/10/be-your-own-ca/) or you could use [etcd-ca](https://github.com/coreos/etcd-ca) to generate certs and keys. 
 
-We need to create our drop-in unit in `/run/systemd/system/etcd.service.d/`. If you run `systemctl status etcd` you can see that CoreOS is already generating a few drop-in units for etcd as part of the OEM and cloudinit processes. To ensure that our drop-in runs after these, we name it `30-certificates.conf`
+We need to create our drop-in unit in `/etc/systemd/system/etcd.service.d/`. If you run `systemctl status etcd` you can see that CoreOS is already generating a few drop-in units for etcd as part of the OEM and cloudinit processes. To ensure that our drop-in runs after these, we name it `30-certificates.conf` and place them in `/etc/`.
```
Comment (crawford) [inline distributed-configuration/customize-etcd-unit/index.md]: might as well make `/etc/` -> `/etc/systemd/system/etcd.service.d/`

Comment (robszumski) [inline distributed-configuration/customize-etcd-unit/index.md]: yeah its a little longer/more wordy but will make it super clear

**Comment by crawford** [comment]
LGTM

**Pull request merged**